tests/common/widgets/toggle-refinement/options.ts

```diff
@@ -18,6 +18,14 @@ function normalizeSnapshot(html: string) {
       .replace(/name="\w+"/g, '')
       .replace(/value="\w+"/g, '')
       .replace(/<input.+?>(\s+)/gs, (match) => match.trim())
+      // Vue InstantSearch adds the count after the label if available.
+      // @MAJOR: Remove this once Vue InstantSearch aligns with spec.
+      .replace(
+        /<span\n?\s*class="ais-ToggleRefinement-count"\n?\s*>\n?.*\s*<\/span>/g,
+        ''
+      )
+      // React renders `checked` as `checked=""` through an attribute, other frameworks use the property.
+      .replace(/checked=""/g, '')
```

Comment:
Would it make sense to move this to the common normalizer?

Reply:
not sure, you need to be aware to use the `toBeChecked` or similar check in the assertion, so I think it may be logical to leave that in the test itself